ubuntu_dockerfile

```diff
@@ -1,219 +1,219 @@
-#
-# STEP 3
-# Run pip install future
-#
-# STEP 4
-# Compile the frontend tests and then run the tests:
-#
-#   node .\node_modules\typescript\bin\tsc --project .
-#   node .\node_modules\karma\bin\karma start .\core\tests\karma.conf.ts
-#
-# If this runs correctly, you will see a SIGKILL at the end. That is okay.
-# If this does not run correctly, visit:
-#   https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Windows%29
-
-##########################################################################
-
-FROM ubuntu:latest
-
-# To bypass tzdata configuration. 
-ENV DEBIAN_FRONTEND noninteractive
-ENV TZ "UTC"
-
-# Install packages needed in Dockerfile
-RUN apt-get update && \
-   apt-get install -y sudo && \
-   apt-get install -y vim && \
-   apt-get install -y wget && \
-   apt-get install -y nodejs && \
-   apt-get install -y npm && \
-   apt-get install -y yes && \
-   apt install -y tzdata && \
-   apt-get clean
-
-# Create oppia directory in Docker container
-RUN mkdir /home/oppia
-
-# Install prerequisites. The yes package responds "yes" to all prompts.
-COPY ./scripts/install_prerequisites.sh /home/oppia/scripts/
-RUN yes | bash /home/oppia/scripts/install_prerequisites.sh
-
-# Fix certificate issues
-RUN apt-get update && \
-   apt-get install ca-certificates-java && \
-   apt-get clean && \
-   update-ca-certificates -f
-
-# Setup JAVA_HOME
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
-ENV PATH $PATH:$JAVA_HOME/bin
-RUN export JAVA_HOME
-
-# Install dumb-init (Signal handling of SIGINT/SIGTERM/SIGKILL and all other unhandled signals)
-# When a process is executed in a Docker container, it is given a PID of 1. Linux kernels give
-# special permissions to the process with a PID of 1. In particular, if the process does not handle
-# a specific signal, then it will not respond to that signal. Unhandled signals for processes of
-# PID other than 1 are set to the default response for the signal. So, we install dumb-init as a
-# reliable process for the Linux kernel to give special permissions to. dumb-init then executes all
-# other processes as child processes (PID other than 1), such that their unhandled signals are
-# automatically handled as expected.
-RUN wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64.deb
-RUN dpkg -i dumb-init_*.deb
-ENTRYPOINT ["dumb-init"]
-
-# Install packages needed for Google Chrome
-RUN apt-get update && \
-   apt-get install -y fonts-liberation libappindicator3-1 libgtk-3-0 libxss1 lsb-release xdg-utils
-
-# Install Google Chrome for frontend tests
-RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-RUN sudo dpkg -i google-chrome-stable_current_amd64.deb
-
-# Copy oppia files into container
-COPY . /home/oppia/
-
-# Allow docker to have sudo privileges
-RUN useradd -m docker && echo "docker:docker" | chpasswd && adduser docker sudo
-USER docker
-
-WORKDIR /home/oppia/
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Why is this file here?

Reply:
This file had CRLF endings before which I changed to LF endings.

Reply:
Ah, okay.